_targets.R

```diff
@@ -5,19 +5,61 @@ library(tibble)
 suppressPackageStartupMessages(library(dplyr))
 
 options(tidyverse.quiet = TRUE)
-tar_option_set(packages = c("tidyverse", "dataRetrieval", "urbnmapr", "rnaturalearth", "cowplot", "lubridate"))
+tar_option_set(packages = c("tidyverse", "dataRetrieval", "urbnmapr",
+                            "rnaturalearth", "cowplot", "lubridate",
+                            "leaflet", "leafpop", "htmlwidgets"))
 
 # Load functions needed by targets below
 source("1_fetch/src/find_oldest_sites.R")
 source("1_fetch/src/get_site_data.R")
 source("2_process/src/tally_site_obs.R")
+source("2_process/src/summarize_targets.R")
 source("3_visualize/src/map_sites.R")
 source("3_visualize/src/plot_site_data.R")
+source("3_visualize/src/plot_data_coverage.R")
+source("3_visualize/src/map_timeseries.R")
 
 # Configuration
 states <- c('WI','MN','MI','IL','IN','IA')
 parameter <- c('00060')
 
+# Static branching targets
+# PULL SITE DATA
+mapped_by_state_targets <- tar_map(
+  values = tibble(state_abb = states,
+                  state_plot_files = sprintf('3_visualize/out/timeseries_%s.png', state_abb)),
+  names = state_abb,
+  unlist = FALSE,
```

Comment:
I like that you put `names` and `unlist` at the top! I have not been doing that and it feels more logical.